### PR TITLE
Add comments and fix a typo

### PR DESCRIPTION
**Pull request by matt-neal**
Here you go Fosty

### CODE REVIEW

File: commands.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def inner(func, description=description):
 
 """
 Each @is_command function is a command (!!), callable by sending "!<function_name>" in chat.
-All replies will be sent in the bot's colour, using /me unless specified otherwise.
+All replies will be sent in the bots colour, using /me unless specified otherwise.
```
Comment (AcesFullOfKings) [inline commands.py]: ????????? This was correct :O

File: commands.py [hunk 2]
```diff
@@ -1168,6 +1168,7 @@ def worldday(message_dict):
 	page = requests.get("https://www.daysoftheyear.com/").text
 
 	# flasgod don't judge me, I know this is wonky af
+	# don't worry, I'm juding you enough for both of us -Moldar
```
Comment (AcesFullOfKings) [inline commands.py:1171]: I do feel judged

File: commands.py [hunk 3]
```diff
@@ -1998,6 +1999,7 @@ def crypto(message_dict):
 	doge(message_dict)
 	log(f"Sent Crypto prices to {user}")
 
+# Please, nobody copy this or use this...it's terrifying.
```
Comment (AcesFullOfKings) [inline commands.py:2002]: Everyone should do this. So convenient.

**Review by AcesFullOfKings — CHANGES_REQUESTED** [review]

File: commands.py (hunk 2 above)
Comment (AcesFullOfKings) [reply commands.py:1171]: THEY DON'T HAVE AN API OK

File: commands.py (hunk 1 above)
Comment (matt-neal) [reply commands.py]: Eh, I just smacked the squiggly blue line from VSC on this one, you right though.

Comment (AcesFullOfKings) [inline commands.py]: wait I did it wrong

**Review by AcesFullOfKings — APPROVED** [review]

**Pull request merged**